iobroker/umbrel-app.yml

```
@@ -0,0 +1,49 @@
+manifestVersion: 1.1
+id: iobroker
+category: automation
+name: ioBroker
+version: '2025.6.1'
+tagline: Open source home automation platform for every use case
+description: >-
+     🧠 **ioBroker** is the Swiss Army knife of home automation.
+
+     With support for **hundreds of devices and protocols**, including Zigbee, Z-Wave, KNX, MQTT, EnOcean, Shelly, Philips Hue, Sonos, Alexa, and more – ioBroker gives you full control over your smart home from a clean, browser-based dashboard.
+
+     💡 Want automation? Want custom dashboards? Want privacy-first logic without cloud lock-in? You've found your home base. ioBroker runs **entirely local** on your Umbrel node. Your data stays yours — 100%.
+
+     --- 🔧 **Features at a glance:**
+
+     - 🌐 Sleek web UI to configure devices and automations
+     - 🧩 400+ adapters and integrations from the ioBroker community
+     - 🛜 Local-first architecture (no vendor cloud required)
+     - 🔐 Full control of your data, locally encrypted
+     - 🧱 Visual logic editor (Blockly/JavaScript-based flows)
+
+     --- 🚀 **Setup instructions:**
+
+     - Open the dashboard at port `8097`
+     - On first launch, follow the **setup wizard**
+     - USB devices (Zigbee, etc.) may require manual mapping in Docker
+     - Use the Adapter tab to explore and install integrations
+
+     For power users: install VIS, Node-RED, BackItUp, Alexa support and more — all through the integrated adapter system.
+
+     🛠️ Learn more at [iobroker.net](https://www.iobroker.net) or join the community: [forum.iobroker.net](https://forum.iobroker.net)
+
+developer: ioBroker GmbH
+website: https://www.iobroker.net/
+dependencies: []
+repo: https://github.com/ioBroker/ioBroker
+support: https://forum.iobroker.net
+port: 8097
```

Use a different port as 8097 is already used in `blockstream-blind-oracle` app
```suggestion
port: 8098
```